Source/Scene/SkyAtmosphere.js

```diff
@@ -96,6 +121,12 @@ define([
         this._command.uniformMap = {
             cameraAndRadiiAndDynamicAtmosphereColor : function() {
                 return that._cameraAndRadiiAndDynamicAtmosphereColor;
+            },
+            u_hsvShift : function() {
```

Comment:
Make sure to be consistent with the naming, in some places it's `hsv` and other `hsb`. Either is fine to use. Changing the uniform name to `hueSaturationValue` or `hueSaturationBrightness` is probably clearer.